cli/cli_app.go

```diff
@@ -534,18 +534,18 @@ func processHooks(hooks []config.Hook, terragruntOptions *options.TerragruntOpti
 
 	errorsOccurred := []error{}
 
-	terragruntOptions.Logger.Printf("Detected %d Hooks", len(hooks))
+	terragruntOptions.Logger.Debugf("Detected %d Hooks", len(hooks))
 
 	for _, curHook := range hooks {
 		allPreviousErrors := append(previousExecError, errorsOccurred...)
 		if shouldRunHook(curHook, terragruntOptions, allPreviousErrors...) {
-			terragruntOptions.Logger.Printf("Executing hook: %s", curHook.Name)
+			terragruntOptions.Logger.Infof("Executing hook: %s", curHook.Name)
 			actionToExecute := curHook.Execute[0]
 			actionParams := curHook.Execute[1:]
 			possibleError := shell.RunShellCommand(terragruntOptions, actionToExecute, actionParams...)
```

Comment:
How is the log output handled by `RunShellCommand`? Are we hiding the logs of the hook scripts being run here?

Reply:
@brikis98 we are returning errors from [RunShellCommandWithOutput()](https://github.com/amnk/terragrunt/blob/amnk/change-default-log-level/shell/run_shell_cmd.go#L52) from what I can see

Reply:
But it's not just the final error, but also `stdout` / `stderr` for the shell command itself, right?

Reply:
Any update on whether this works correctly?

Reply:
This particular change only affects log line itself. I tested hook output, and it is shown under `Info`, so all is fine?

Reply:
As long as hook output is visible, that sounds good to me!